app/labor/assign_tag_moderator.rb

```diff
@@ -17,7 +17,7 @@ def self.add_tag_moderators(user_ids, tag_ids)
       add_tag_mod_role(user, tag)
       add_trusted_role(user)
       add_to_chat_channels(user, tag)
-      NotifyMailer.tag_moderator_confirmation_email(user, tag).deliver unless tag.name == "go"
```

Comment:
Go was the first tag that we added multiple tag mods too. We initially stopped the welcome email from going out for Go Mods b/c we thought the welcome mail might be confusing. Hence, this can be removed now.
https://dev-internal.slack.com/archives/C7GE84DEJ/p1587112951346400